### ISSUE TITLE
Can't set Ctrl+Backspace in Terminal to deleteWordLeft

### ISSUE THREAD

**Issue by ktruong19** [Microsoft/vscode#19697]
- VSCode Version: 1.8.1
- OS Version: windows 10

Steps to Reproduce:

1. Open terminal
2. Enter multiple words/commands
3. Press Ctrl+Backspace to delete full word left like it does in editor. Notice does not work.
4. Open Key Bindings to try to set, notice no available options for terminal.

**Comment by Tyriar** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/11314

As a workaround you should be able to use ctrl+w for this right now.

**Comment by ktruong19** [comment]
Hi,

Ctrl + w does not work for me.

Kenyth

**Comment by Tyriar** [comment]
Ah you're on Windows, this may be a limitation on Windows even after #11314, unless your using a shell like Git Bash in which ctrl+w should work.
